config/betterquesting/DefaultQuests.json

```diff
@@ -306452,6 +306452,415 @@
           }
         }
       }
+    },
+    "2897:10": {
+      "questID:3": 2897,
+      "preRequisites:11": [
```

Comment:
I can't tell what tab this is on, but hopefully it's on the tab with all the weapons? And has the appropriate pre-reqs? Like, bronze and/or the tinker table? I'm not sure why there's 3.

Reply:
explain i dont understand whyt you mean? i make a hidden quest if you grab the flippers it will be open like the one ring

Reply:
This is the rapier quest though? What does it have to do with flippers? I collected all but the first few weapon quests on the same tab.

Reply:
ok you mean we have a rapier quest allready ?

Reply:
That's not what I said at all. I'm saying weapon quests go on that tab, in case you put it somewhere else.

Reply:
i put it beside the sword (bronze) and i guess this is in this case ok. I can add it to the stop from dying page as well

Reply:
You can't even get to the TF in steam though?

Reply:
and this quest is in steam age

Reply:
yes ik but it is a side quest and you can get it when you have bronze. You can make a bow in steam age too.

Reply:
added to the tab